### PR TITLE
Fix broken custom runtimes link

### PR DESCRIPTION
**Pull request by edshee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ you interact with a subset of common frameworks.
 This allows you to start serving models saved in these frameworks straight
 away.
 However, it's also possible to **write [custom
-runtimes](./docs/runtimes/custom)**.
```
Comment (adriangonz) [inline README.md:68]: Great spot!

I think this should be pointing to https://github.com/SeldonIO/MLServer/blob/master/docs/runtimes/custom.md instead, which is meant as the "homepage" for custom runtimes.

**Review by adriangonz — COMMENTED** [review]
Nice catch @edshee !

File: README.md (hunk 1 above)
Comment (edshee) [reply README.md:68]: Yep you're right, I linked the examples one instead

**Comment by edshee** [comment]
Leaving for you to merge just for the sake of process :nerd_face:

**Pull request merged**